clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/description/manifest/KubernetesManifestStrategy.java

```diff
@@ -107,7 +107,8 @@ ImmutableMap<String, String> toAnnotations() {
   public enum DeployStrategy {
     APPLY(null),
     RECREATE(STRATEGY_ANNOTATION_PREFIX + "/recreate"),
-    REPLACE(STRATEGY_ANNOTATION_PREFIX + "/replace");
+    REPLACE(STRATEGY_ANNOTATION_PREFIX + "/replace"),
+    SERVER_SIDE_APPLY(STRATEGY_ANNOTATION_PREFIX + "/server-side-apply");
```

Comment:
From https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts, it seems worth trying to support `--force-conflicts`.

Reply:
what is your recommendation? as another strategy annotation?

this can get tricky with fields such as `replicas` where HPA can become the manager, but force could take over.

Reply:
I was thinking of using a value for this annotation, something like `strategy.spinnaker.io/server-side-apply: force` to indicate `--force-conflicts`.

Reply:
took a quick look and it may require some refactoring since some code paths assume the value to be a boolean.

would `strategy.spinnaker.io/server-side-apply-force-conflicts` be a good option here?

Reply:
@dbyron-sf anything I can do to get this PR moving?

Looking at Armory scale agent, it seems to only support `serverside-apply` flag but it seems to add force-conflicts by default. Thoughts?

Reply:
Sorry, I've been swamped lately.  I'm pretty sure
```
strategy.spinnaker.io/server-side-apply: force
```
for force, and...I dunno, for the no-force case, maybe
```
strategy.spinnaker.io/server-side-apply: no-force
```
I believe adding something like this (untested):
```
  public enum ServerSideApplyStrategy {
    FORCE(ImmutableMap.of(SERVER_SIDE_APPLY_STRATEGY, "force")),
    NO_FORCE(ImmutableMap.of(SERVER_SIDE_APPLY_STRATEGY, "no-force")),
    DEFAULT(ImmutableMap.of());
    private final ImmutableMap<String, String> annotations;

    ServerSideApplyStrategy(ImmutableMap<String, String> annotations) {
      this.annotations = annotations;
    }

    static ServerSideApplyStrategy fromAnnotations(Map<String, String> annotations) {
      if (annotations.containsKey(SERVER_SIDE_APPLY_STRATEGY)) {
        switch (annotations.get(SERVER_SIDE_APPLY_STRATEGY)) {
          case "force":
            return FORCE;
          case "no-force":
            return NO_FORCE;
          default:
            return DEFAULT;
        }
      }
      return DEFAULT;
    }

    ImmutableMap<String, String> toAnnotations() {
      return annotations;
    }
  }
```
and a
```
private final ServerSideApplyStrategy serverSideApplyStrategy;
```
member to KubernetesManifestStrategy, initialized with:
```
    this.serverSideApplyStrategy =
        Optional.ofNullable(serverSideApplyStrategy).orElse(ServerSideApplyStrategy.DEFAULT);
```
 with a call to
```
    builder.putAll(serverSideApplyStrategy.toAnnotations());
```
in toAnnotations and
```
        .serverSideApplyStrategy(ServerSideApplyStrategy.fromAnnotations(annotations))
```
works.